k8s/demo/cluster-00/bsp/bulk-scan-orchestrator.yaml

```diff
@@ -22,7 +22,6 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/bulk-scan/orchestrator:prod-c035f888
       environment:
-        TRANSFORMATION_URL_SSCS: http://sscs-bulk-scan-demo.service.core-compute-demo.internal
```

Comment:
why do we need to remove it from demo ?

Reply:
or your updating processor values yaml ?

Reply:
we don't need it after this change https://github.com/hmcts/bulk-scan-processor/pull/1328

Reply:
I thought of raising a separate PR for version. can do in this PR

Reply:
so then we need chart bump in this PR. So first we need to merge PR 1328 and then this should have version bump change.
Also worth checking when is SSCS going to prod as we may just remove empty strings for prod

Reply:
same goes for orchestrator